src/client/components/ShoppingItem/ShoppingItem.js

```diff
@@ -10,32 +10,47 @@ export default function ShoppingItem({
   productName,
   quantity,
   price,
+  initValue,
+  isDisable,
 }) {
   const [itemValue, setItemValue] = useState(1);
+  const textColor = isDisable ? '#d3d3d3' : 'black';
   return (
     <div className="shopping-item">
       <img src={productImg} alt="" width="185px" height="145px" />
       <div className="item-settings">
         <h2>{productName}</h2>
-        <p>({quantity} units left)</p>
+        <p style={{ color: textColor }}>({quantity} units left)</p>
         <div className="trash">
-          <button type="button" onCLick="" className="remove">
-            <Trash height="20" /> Remove Item
+          <button
+            disabled={isDisable}
+            type="button"
+            onCLick=""
+            className="remove"
+          >
+            <Trash
+              height="20"
+              stroke={isDisable ? '#d3d3d3' : '#000000'}
+              fill={isDisable ? '#d3d3d3' : '#000000'}
+            />{' '}
```

Comment:
Semih , maybe good to remove this space.

Reply:
Done :+1: